docs/source/sctool/task.rst

```diff
@@ -17,21 +17,23 @@ The task command set allows you to schedule, start, stop and modify tasks.
 
```

Comment:
@mmatczuk According to the YAML files, now  ``sctool task <command> [flags] [global flags]`` is ``sctool <command> [flags] [global flags]``, right?

Reply:
Yes.